src/main/java/com/kainos/ea/dao/JobsDao.java

```diff
@@ -12,7 +12,7 @@
 public class JobsDao {
     public List<Job> getJobs(Connection c) throws SQLException, DatabaseConnectionException {
         Statement st = c.createStatement();
-        PreparedStatement statement = c.prepareStatement("SELECT id, name, job_description FROM job_roles");
+        PreparedStatement statement = c.prepareStatement("SELECT id, name, job_description, job_responsibilities FROM job_roles");
```

Comment:
Perhaps the prefix of "job_" in the job_roles table is unnecessary? How about changing it to description and responsibilities.